### PR TITLE
🔧 upgrade nuxt-edge

### PR DESCRIPTION
**Pull request by roiLeo**
Will it blend?

Capture d’écran 2023-08-16 à 1 59 37 PM

### CODE REVIEW

**Comment by roiLeo** [comment]
🤔 it does build, I wasn't expecting that

**Comment by preschian** [comment]
Are `nuxt-edge` and `nuxt@2x` the same thing? they change it since this commit https://github.com/nuxt/nuxt/pull/18836/files

For me, the problem was if I upgraded `@nuxt/bridge` to the latest version

But, I hope you can make it

File: nuxt.config.js [hunk 1]
```diff
@@ -322,6 +322,16 @@ export default defineNuxtConfig({
   },
 
   hooks: {
+    ready(nuxt) {
+      // https://github.com/nuxt/bridge/issues/607
+      // translate nuxt 2 hook from @nuxt/webpack-edge to nuxt bridge hook
+      nuxt.hook('server:devMiddleware', async (devMiddleware) => {
```
Comment (preschian) [inline nuxt.config.js:328]: let's remove this file https://github.com/kodadot/nft-gallery/blob/main/patches/%40nuxt__webpack-edge%402.16.0-27720022.54e852f.patch

**Comment by roiLeo** [comment]
> Are `nuxt-edge` and `nuxt@2x` the same thing? they change it since this commit https://github.com/nuxt/nuxt/pull/18836/files

Idk about that just saw a [new version](https://www.npmjs.com/package/nuxt-edge?activeTab=versions) of nuxt-edge being push

> For me, the problem was if I upgraded `@nuxt/bridge` to the latest version

Did you remember what was the issue, I'll try to upgrade to current version

**Comment by preschian** [comment]
> Did you remember what was the issue, I'll try to upgrade to current version

I think you could try in a separate branch

1. https://github.com/preschian/nft-gallery/pull/86 -> build was fine, error in the browser -> `useLocalStorage is not defined`
2. https://github.com/preschian/nft-gallery/pull/96 -> build was fine, error in the browser -> `Cannot read properties of undefined (reading 'provide')`

both of them was blank page

**Comment by roiLeo** [comment]
> I think you could try in a separate branch

Aight, then sould I merge this one?

**Comment by preschian** [comment]
> Aight, then sould I merge this one?

yes, I think this one is safe to merge. In case something happens, just use `nuxt@2x`

**Comment by yangwao** [comment]
I sense merge

https://github.com/kodadot/nft-gallery/assets/5887929/b97316be-8265-4f25-af03-73eeb26bb46d

**Comment by roiLeo** [comment]
dinamit-bom

**Comment by preschian** [comment]
> Woah so many issue with `@nuxt/bridge@npm:@nuxt/bridge-edge`

iirc, error since they upgrade nitro to v2 https://github.com/nuxt/bridge/pull/695/files
maybe there is some nitro config that I missed

**Pull request merged**